client/src/audio.c

```diff
@@ -328,8 +328,7 @@ void audio_playbackStart(int channels, int sampleRate, PSAudioFormat format,
     playbackStop();
 
   int srcError;
-  audio.playback.spiceData.src =
-    src_new(SRC_SINC_BEST_QUALITY, channels, &srcError);
+  audio.playback.spiceData.src = src_new(SRC_SINC_FASTEST, channels, &srcError);
```

Comment:
Line too long, please limit to 80 columns.

Reply:
The line is exactly 80 characters wide. Do you prefer limiting it to 79?

Reply:
doh, sorry about that, you're correct.